README.md

```diff
@@ -503,7 +503,7 @@ Based on this specification `krane run` will create a new pod with the entrypoin
 
 *Options:*
 
-* `--template=TEMPLATE`:  Specifies the name of the PodTemplate to use (default is `task-runner-template` if this option is not set).
+* `--template=TEMPLATE`: Specifies the name of the PodTemplate to use.
```

Comment:
Wasn't sure what the convention was to mention the option is required.

I was going to put:

```suggestion
* `--template=TEMPLATE` (required): Specifies the name of the PodTemplate to use.
```

Reply:
Not a convention we're currently using.